OptimizelySDK/OptimizelyUserContext.cs

```diff
@@ -14,18 +14,29 @@
  * limitations under the License.
  */
 
-using OptimizelySDK.Logger;
-using System.Collections.Generic;
+#if !(NET35 || NET40 || NETSTANDARD1_6)
+#define USE_ODP
```

Comment:
I see this #define in several places. It looks like not shared in C#. Can we also consider to open ODPManager to all and use NoOpODPManager for NET35, which you already defined?

Reply:
To move this PR forward, I'd like to handle conditional implementation in older frameworks at a later time or when we move to modernizing this SDK.